site/docs/guidelines/forms.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Forms
+navTitle: Forms
+summaryParagraph: Use forms to allow people to enter data for use by the application, or to configure options. Designing forms well requires making decisions about structure, sequence, interface elements, field labels, help, and feedback. 
+  Alignment creates visual connections, a sense of order, and hierarchy. Horizontal rhythm keeps horizontal spaces between elements on a page consistent.
```

Comment:
I'm not sure what happened here, it seems we have a mash up on the alignment page description and the forms description. I think we can shorten it a bit too. How about "Forms let people enter data needed to use the platform."?

Reply:
Good call, will update 👍